File: SCUnreadMenu/SCAppDelegate.m
//
//  SCAppDelegate.m
//  SCUnreadMenu
//
//  Created by Sam Page on 16/03/14.
//  Copyright (c) 2014 Subjective-C. All rights reserved.
//

#import "SCAppDelegate.h"
#import "SCUnreadMenu-Swift.h"

@implementation SCAppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
    self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];

    self.rootViewController = [[SCRootViewController alloc] initWithNibName:nil bundle:nil];
    self.window.rootViewController = _rootViewController;
    
    [self.window makeKeyAndVisible];
    return YES;
}

@end

File: SCUnreadMenu/SCAppDelegate.swift
//
//  SCAppDelegate.swift
//  SCUnreadMenu
//
//  Created by Yichi on 27/02/2015.
//  Copyright (c) 2015 Subjective-C. All rights reserved.
//

import UIKit

@UIApplicationMain @objc
class SCAppDelegate : UIResponder, UIApplicationDelegate {
	lazy var window:UIWindow = {
		let window = UIWindow(frame: UIScreen.mainScreen().bounds)
		window.rootViewController = self.rootViewController
		return window
	}()
	lazy var rootViewController:SCRootViewController = {
		return SCRootViewController(nibName: nil, bundle: nil)
	}()
	
	func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject : AnyObject]?) -> Bool {
		
		self.window.makeKeyAndVisible()
		return true
	}
}